doc/users/whats_new.rst

```diff
@@ -86,6 +86,13 @@ wrap around, they are a good choice for cyclic data such as phase
 angles, compass directions, or time of day. Like *viridis*, *twilight* is
 perceptually uniform and colorblind friendly.
 
+Add Solarized palette to named matplotlib colors
```

Comment:
Can you put this in it's own file per https://github.com/matplotlib/matplotlib/blob/master/doc/users/next_whats_new/README.rst ?